### ISSUE TITLE
Make a v1 of blog static snapshotting at blog-snapshot-beta.openhatch.org

### ISSUE THREAD

**Issue by paulproteus** [openhatch/winddown#4]
Then get feedback on it.

Risky aspects:

- Make sure to include all wp-content assets.

- Make sure links seem to work. (Put it through a broken link checker?)

Other:

- It'd be nice to add a note to the top of the blog somewhere indicating that we've wound-down.

**Comment by paulproteus** [comment]
Current status:

- blog-snapshot-beta.openhatch.org exists in our DNS and points at this GitHub repo: https://github.com/paulproteus/blog.openhatch.org

- @ehashman , are you possibly willing to review it?

**Review**

I'm looking for:

- Are there any technical issues here that you can find, that bother you?

- If this stays online for 10 years, will it basically make sense? (although I know I need to design where the "we closed-down" banner goes)

**Post-review**

After it passes review, my plan is to revert commit 55eea4867c181840921c770a7f676fc40e1a7322 and then update DNS accordingly. I'll probably use CloudFlare to configure this as an always-HTTPS zone, just for the fun of it.

**Comment by paulproteus** [comment]
+1, done.

**Comment by ehashman** [comment]
The static backup looked fine to me at a glance, fwiw. I was not sure on the best way to test it, so I didn't get that far.

**Comment by paulproteus** [comment]
Thanks for testing it! :D 

BTW, it's hosted on my github pages; I'm using cloudflare for free-of-cost DNS & HTTPS; the top banner is arguably too tall but I wanted space on mobile. I'll probably use the same design on the main OpenHatch site.

Also there's a backup of the original MySQL + PHP here:

https://github.com/paulproteus/blog.openhatch.org/tree/master/wpengine-site-backup

You'll see a database snapshot here:

https://github.com/paulproteus/blog.openhatch.org/blob/master/wpengine-site-backup/wp-content/mysql.sql

I removed the users table, and I closed OpenHatch's WP-Engine account.

@ehashman thank you greatly for leaving a note! :)
